RELEASE.md

```diff
@@ -1,33 +1,68 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+All pull requests should be made from a feature branch. Never push
+directly to `master`.
+
+If your pull request will change any behavior
+of the plugin (including bug fixes and new features), you should
+update the version string. When changing only documentation
+(e.g. this file or the README), do not update the version string.
+
+After you have made the changes you wish to merge into `master`,
+you should:
+1. Choose a new version string. Please try to respect
+[semantic versioning](https://semver.org/). Do not augment the major
+version without explicit approval from all the maintainers.
+Try to avoid version numbers that will conflict with 
```

Comment:
This is not actionable.  It is also a bad idea, because there is no knowing what order pull requests will be merged.